Documentation/environment-variables.rst

```diff
@@ -7,28 +7,27 @@ Communication Thread
 --------------------
 
 By default, any SeisSol run with more than one MPI rank will use a communication thread to advance the MPI progress engine.
-For that, you will need to leave at least one thread vacant in your OpenMP thread placing map, cf. the SuperMUC-NG example below.
+For that, you must leave at least one thread vacant in your OpenMP thread placing map, cf. the SuperMUC-NG example below.
 
-If you do not want to use a communication thread, you may set `SEISSOL_COMMTHREAD=0`; then SeisSol polls on the progress from time to time.
+If you do not want to use a communication thread, you may set `SEISSOL_COMMTHREAD=0`; then, SeisSol polls the progress from time to time.
 
 Load Balancing
 --------------
 
-When running with multiple ranks, SeisSol will estimate the performance of a node, to enable better load balancing for it.
-For that, it runs the so-called "Mini SeisSol" benchmark. As its name already hints at, it simulates a small test workload on each node;
+When running with multiple ranks, SeisSol will estimate a node's performance, enabling better load balancing.
+For that, it runs the so-called "Mini SeisSol" benchmark. As its name already hints, it simulates a small test workload on each node;
 thus estimating the performance of all nodes relative to each other. The number of elements per node assigned during the partitioning will be resized according to these values.
 
-As a result, the partitioning of runs may become non-deterministic, and the initialization procedure may take a little longer; especially when running only on a single node with multiple ranks.
+As a result, the partitioning of runs may become non-deterministic, and the initialization procedure may take a little longer, especially when running only on a single node with multiple ranks.
 To disable it, set `SEISSOL_MINISEISSOL=0`.
 
 Persistent MPI Operations
 -------------------------
 
-Since SeisSol has a static communication pattern (in the sense of: per iteration, we issue the same MPI transfer requests),
-we may use persistent MPI communication—it may reduce the communication latency.
+Since SeisSol has a static communication pattern (in the sense that per iteration, we issue the same MPI transfer requests). 
```

Comment:
The sentence stop here will cause the sentence to sound a bit incomplete. We could e.g. remove the bracket, going for something like "SeisSol has a static communication pattern. That means, we issue the same MPI transfer requests in each iteration. Thus, we use persistent MPI [...]".

Reply:
Modified but within the same sentence